src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/__snapshots__/GrammarTests.test.ts.snap

```diff
@@ -5169,8 +5169,7 @@ Line:             Below is an incomplete tag
 
 Line:             </strong>
  - token from 0 to 12 (            ) with scopes text.aspnetcorerazor, meta.structure.razor.codeblock, source.cs
- - token from 12 to 14 (</) with scopes text.aspnetcorerazor, meta.structure.razor.codeblock, source.cs, punctuation.definition.tag.begin.html
- - token from 14 to 20 (strong) with scopes text.aspnetcorerazor, meta.structure.razor.codeblock, source.cs, entity.name.tag.html
+ - token from 12 to 20 (</strong) with scopes text.aspnetcorerazor, meta.structure.razor.codeblock, source.cs
```

Comment:
At first  this was concerning; however, after closer inspection it looks like it's an incomplete tag in a codeblock